app/services/hyrax/workflow/workflow_importer.rb

```diff
@@ -1,27 +1,41 @@
 module Hyrax
   module Workflow
     class WorkflowImporter
-      # @api public
-      #
-      # Load all the workflows in config/workflows/*.json
-      # @return [TrueClass]
-      def self.load_workflows
-        Dir.glob(Rails.root + "config/workflows/*.json") do |config|
-          Rails.logger.info "Loading workflow: #{config}"
-          generate_from_json_file(path: config)
+      class << self
```

Comment:
Moving all of the methods into a `class << self` and removing the `self.` part of method definition has made reviewing the changes more difficult. Please preserve the original methods and add the `:load_errors` accessor via the following:

```ruby
class << self
  attr_accessor :load_errors
end
```

One side-effect of the :load_errors is that they will remain for the duration of the script. If these are loaded via the Rails UI (e.g. by allowing JSON files to be uploaded and load the WorkflowImporter), the load_errors will not clear out.